src/app/dataset-flow/dataset-flow-details/tabs/flow-details-summary-tab/flow-details-summary-tab.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="container">
+    <div class="row">
+        <div class="col pl-40">
+            <div class="row mb-4">
+                <div class="col-md-3"><span class="fw-bold">Flow type</span></div>
+                <div class="col-md-9" data-test-id="flow-type">{{ flowTypeDescription(flowDetails) }}</div>
+            </div>
+            <div class="row mb-4">
+                <div class="col-md-3"><span class="fw-bold">Status</span></div>
+                <div class="col-md-9" data-test-id="flow-status">{{ flowDetails.status | titlecase }}</div>
+            </div>
+            <div class="row mb-4">
+                <div class="col-md-3">
+                    <span class="fw-bold">Initiator</span>
+                </div>
+                <div class="col-md-9" data-test-id="flow-initiator">
+                    {{ flowDetails.initiator?.accountName ?? "System" }}
```

Comment:
Make sure we have this default name in 1 single place. I think in other place we used text "System Process". We should have 1 version, and it should be in `AppValues`

Reply:
Fixed